docs/comunidad/canales-de-telegram.md

```diff
@@ -27,13 +26,13 @@ Los canales de Telegram son una herramienta ideal para difundir mensajes a grand
 - [EOS Opportunities](https://t.me/eos_opportunities)
 - [Jungle Testnet](https://t.me/jungletestnet)
 
-### Comunidad EOS Costa Rica
+## Comunidad EOS Costa Rica
 
 * [LatamLink](https://t.me/latamlink)
 * [EOS Surf](https://t.me/eosbeach)
 * [Blockchain Costa Rica](https://t.me/blockchaincostarica)
 
-### Comunidad Desarrolladores
+## Comunidad Desarrolladores
```

Comment:
"Comunidad de Desarrolladores"?

Reply:
I didn't see this mistake, good observation